test/Index.spec.ts

```diff
@@ -115,14 +239,14 @@ describe('Collection Api', () => {
     expect(res.status.error_code).toEqual(ErrorCode.SUCCESS);
   });
 
-  // it(`Get Index without name progress`, async () => {
-  //   const res = await milvusClient.indexManager.getIndexBuildProgress({
-  //     collection_name: COLLECTION_NAME_WITHOUT_INDEX_NAME,
-  //     index_name: INDEX_NAME,
-  //   });
-  //   // console.log('----getIndexBuildProgress with name ----', res);
-  //   expect(res.status.error_code).toEqual(ErrorCode.SUCCESS);
-  // });
+  it(`Get Index without name progress`, async () => {
```

Comment:
this test should test get progress without index name, and use default index name in Milvus.
May be we don‘t need it anymore?